src/shared/views/map/MapTab/MapTab.js

```diff
@@ -1,11 +1,81 @@
 import React from 'react'
```

Comment:
Awesome nice work again. I would say eventually we'll take this out and make it generic and then use it on the map page.

That way we get to use the controls you created everywhere.

Look and works really nicely!